tests/service_controller_test.go

```diff
@@ -0,0 +1,52 @@
+package tests
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	"kubevirt.io/ssp-operator/controllers"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func getSspMetricsService() (*v1.Service, error) {
+	service := controllers.ServiceObject(strategy.GetNamespace())
+	err := apiClient.Get(ctx, client.ObjectKeyFromObject(service), service)
+	return service, err
+}
+
+var _ = Describe("Service Controller", func() {
+	var (
+		err     error
+		service *v1.Service
+	)
+	BeforeEach(func() {
+		waitUntilDeployed()
+
+		service, err = getSspMetricsService()
+	})
+
+	Context("creation", func() {
+		It(fmt.Sprintf("[test_id: ???] Should create %s service", controllers.MetricsServiceName), func() {
+			if err != nil {
```

Comment:
I don't like this design, testing an error from `BeforeEach()` in an `It` test. Can you move the `getSspMetricsService()` call to this test and use `Expect()` ?

Reply:
but we need it twice, i changed the naming a bit, see if it is better now.